packages/zmarkdown/server.js

```diff
@@ -0,0 +1,41 @@
+const toVFile = require('to-vfile')
+const clone = require('clone')
+const rebberStringify = require('rebber/src')
+const zmarkdown = require('./common')
+
+const fromFile = (filepath) => toVFile.readSync(filepath)
+
+const remarkConfig = require('./config/remark')
+let zmd
```

Comment:
It seems to me that nothing ever gets assigned to this variable?

Reply:
I think this is the origin of my bug (test fails).